core/src/test/java/org/elasticsearch/index/fielddata/BinaryDVFieldDataTests.java

```diff
@@ -56,24 +58,24 @@ public void testDocValue() throws Exception {
         bytesList1.add(randomBytes());
         bytesList1.add(randomBytes());
         XContentBuilder doc = XContentFactory.jsonBuilder().startObject().startArray("field").value(bytesList1.get(0)).value(bytesList1.get(1)).endArray().endObject();
-        ParsedDocument d = mapper.parse("test", "test", "1", doc.bytes());
+        ParsedDocument d = mapper.parse(SourceToParse.source("test", "test", "1", doc.bytes(),XContentType.JSON));
```

Comment:
Added a space after the , and XContentType.JSON for all uses other than the ones that had a very large line count, which was separated into another line where the , was done above
e.g. 
`,
XContentType.JSON)`